### ISSUE TITLE
Questing

### ISSUE THREAD

**Issue by TrueGuardian32** [cyberstrawberry101/Omega-Adventure-Pack#15]
If I spent a few hours working on Questing for the pack, would you add it after testing it out?

**Comment by powerhungryring** [comment]
Yes, I will help you when i get done with my semester in college

**Comment by EverSoul** [comment]
Make sure you are in the Discord so you can communicate with us easier.

**Comment by TrueGuardian32** [comment]
Understood, I'll start working on it now

**Comment by TrueGuardian32** [comment]
Can you create a private discord chat where we can talk about quests?
